README.md

```diff
@@ -14,113 +10,19 @@ Welcome to our Mozilla Study Group! A few things to do & know now that you're he
-
-### Check out the Handbook
-
-The instructions below will help you set up the online tools for your Mozilla Study Group - but if you're looking for organziation strategies, event plans and lesson ideas, check out the [Mozilla Study Group Handbook](https://mozillascience.github.io/studyGroupHandbook/)!
-
-## How to Set Up Your Own Mozilla Study Group Website
-
-Everything you need to set up your own Mozilla Study Group website for organizing events is right here - follow the following steps and you'll be up and running soon, and if you have trouble, [open an issue](https://github.com/mozillascience/studyGroup/issues) and we'll help you out!
-
- 1. **Make yourself an account on GitHub.** The free one is just fine.
- 2. **Fork this repository.** Up in the top right corner of this page, there's a button that says 'Fork'; press it! This makes your very own copy of all this stuff in your space on GitHub; when the copy is done, GitHub will automatically take you there.
- 3. **Turn on the Issue Tracker.** An issue tracker is a message board GitHub sets up for every repository; you'll use it to post information about your event and talk to your participants. To turn it on: 
-   - click on 'Settings' in the right sidebar of your repo;
-   - click the checkbox beside 'Issues'; 
-   - head back to your repo by clicking on the big `studyGroup` at the top of the page.
- 4. **Edit the `_config.yml` file** in your new repository:
-   - click on `_config.yml`;
-   - click on the little pencil near the top right;
-   - follow the instructions in the file on how to edit it; 
-   - when you're done, press the green 'Commit Changes' button at the bottom of the page.
-
-That's it, you're done! You can see your new website at `https://yourUserName.github.io/studyGroup/`, where `yourUserName` is the user name you signed up for GitHub with. If this is your first time making a webpage on GitHub, it might take 30 minutes for things to percolate through their computers - don't worry, it's all good, check back later and your website should be up and running.
+ - **Add yourself to the website:** If you'd like to appear on the website under the 'Who we Are' section, have a look at the `_data/members.yml`; send us a pull request with an entry for yourself, or open an issue and we'll do it for you. Basically, we need the following information:
 
-### Optional: Setting up a Google Calendar for your Study Group
-
-If you'd like to offer your community a calendar of events they can import into their own calendars, try using a Google Calendar. To set up, make a new google account, and update the variables in `_config.yml` under the heading 'Setup Google Calendar'.
-
-You can add events to your calendar by hand, but if you'd like to manage it automatically, there's a script to do so in `scripts/updateCalendar.py`; instructions for use are at the top of that file.
-
-## How to Launch a New Event
```

Comment:
Can we keep this in the README or move it to another file @arnavs ? I use it as a reference when I forget how to do this...

Reply:
My thought was that when we needed to do this, we can just check the file history. But whatever you want.

Reply:
Making `admin.md` probably isn't the worst idea.

Reply:
@ttimbers Done.